### PR TITLE
Snake messages

### PR DESCRIPTION
**Pull request by lidldev**
# What issue is this addressing?

### CODE REVIEW

File: examples/snake/main.go [hunk 1]
```diff
@@ -171,11 +175,13 @@ func (g *Game) Update() error {
 }
 
 func (g *Game) Draw(screen *ebiten.Image) {
+	// Here, a snake and an apple are drawn with vector graphics. The range method is used to add behind the snake.
 	for _, v := range g.snakeBody {
 		vector.DrawFilledRect(screen, float32(v.X*gridSize), float32(v.Y*gridSize), gridSize, gridSize, color.RGBA{0x80, 0xa0, 0xc0, 0xff}, false)
 	}
 	vector.DrawFilledRect(screen, float32(g.apple.X*gridSize), float32(g.apple.Y*gridSize), gridSize, gridSize, color.RGBA{0xFF, 0x00, 0x00, 0xff}, false)
 
+	// The if statement is used here to warn the snake with a button to start if it doesn't move.
```
Comment (hajimehoshi) [inline examples/snake/main.go:184]: Remove this comment.

File: examples/snake/main.go [hunk 2]
```diff
@@ -171,11 +175,13 @@ func (g *Game) Update() error {
 }
 
 func (g *Game) Draw(screen *ebiten.Image) {
+	// Here, a snake and an apple are drawn with vector graphics. The range method is used to add behind the snake.
```
Comment (hajimehoshi) [inline examples/snake/main.go:178]: Remove this comment.

File: examples/snake/main.go [hunk 3]
```diff
@@ -102,6 +102,10 @@ func (g *Game) reset() {
 }
 
 func (g *Game) Update() error {
+	// Here, in order to prevent it from suddenly going in the opposite direction in the movement logic,
+	// block the value used to move to the opposite side with an if statement,
+	// for instance:
+	// If the snake suddenly presses the go left button while going to the right, so that it does not go to the left.
```
Comment (hajimehoshi) [inline examples/snake/main.go:108]:
```
// Decide the snake's direction along with the user input.
// A U-turn is forbidden here (e.g. if the snake is moving in the left direction, the snake cannot goes to the right direction immediately).
```

Comment (Zyko0) [reply examples/snake/main.go:108]: cannot go*

Comment (hajimehoshi) [reply examples/snake/main.go:108]: Oh thanks!

```
// Decide the snake's direction along with the user input.
// A U-turn is forbidden here (e.g. if the snake is moving in the left direction, the snake cannot go to the right direction immediately).
```

**Review by hajimehoshi — APPROVED** [review]
LGTM, thanks!

**Pull request merged**